### PR TITLE
ISSUE-640 Apisix - plugin - client control for setting the max size of the client request body

### PR DESCRIPTION
**Pull request by vttranlina**
resolve https://github.com/linagora/james-project/issues/5074

### CODE REVIEW

**Comment by vttranlina** [comment]
Local test
image

File: demo/apisix/conf/apisix.yaml [hunk 1]
```diff
@@ -264,6 +264,8 @@ plugin_configs:
           failures: 3  # should be var: JMAP_CIRCUIT_BREAKER_MAXERRORS
         healthy:
           successes: 1
+      client-control:
+        max_body_size: 20000000
```
Comment (quantranhong1999) [inline demo/apisix/conf/apisix.yaml:268]: Should be lower e.g. 5000000? cf https://github.com/linagora/james-project/issues/5074#issuecomment-1956870172

**Review by quantranhong1999 — COMMENTED** [review]
Would be nice to have a contribution on James APISIX example too IMO.

File: demo/apisix/conf/apisix.yaml (hunk 1 above)
Comment (chibenwa) [reply demo/apisix/conf/apisix.yaml:268]: 1MB for everything.
20MB default for uploads...

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Pull request merged**